recipes/azure-synapse-spark/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "azure-synapse-spark" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/azure-synapse-spark-{{ version }}.zip
+  sha256: 86fa29463a24b7c37025ff21509b70e36b4dace28e5d92001bc920488350acd5
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - msrest >=0.5.0
+    - azure-common >=1.1,<2.dev0
+    - azure-core >=1.6.0,<2.0.0
+    - azure-synapse-nspkg  # [py<30]
```

Comment:
You can probably remove this dependency and make it a `noarch` package that is `py>=3.0`. In conda-forge we don't build 3.0 anyway, we are building 3.7 and up (and we'll dropping 3.7 soon BTW).